src/commands/autocomplete/create.ts

```diff
@@ -11,6 +11,13 @@ type CommandCompletion = {
   flags: any
 }
 
+function sanitizeDescription(description?: string): string {
+  if (description === undefined) {
+    return ''
+  }
+  return description.replace(/`/g, '\\\\\\\`')
```

Comment:
I know it needs 3 backslashes to escape twice, why the other 4? Other than that looks good!

Reply:
Each double backslash becomes a backslash in the end, because of Javascript way of processing strings. The 7th one is to escape the backtick, same reason.

If you run the code, by linking the branch's folder into one of your CLI, you could verify the produced zsh autocompletion file. It will have the 3 backslashs as expected.